### PR TITLE
+ added details for CentOS/RHEL 7

### PR DESCRIPTION
**Pull request by eRadical**
A suite of CentOS 7 servers needed updates for GeoIP2 library... I did that on the servers and made this PR so other can find it easily.

### CODE REVIEW

**Review by Noey — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -26,17 +26,30 @@ Then install `geoipupdate` by running:
...
 Once you have the necessary dependencies, run the following commands:
 
     $ ./configure
     $ make
     $ sudo make install
 
+On CentOS 7 or RHEL 7 to make a compatible build with the GeoIP-1.5.x
+library and config files you need to configure with:
+
+    $ ./configure --prefix=/usr --sysconfdir=/etc
```
Comment (oschwald) [inline README.md]: The first two sections look good, but locally installed software should go in `/usr/local`, not `/usr`. Recommending that people install in `/usr` will lead to conflicts with the system package management. I'd be happy to merge with this section removed.

Comment (eRadical) [reply README.md]: Yes you are right! But I was preaching for a build as a **drop in replacement** for the GeoIP-1.5.x version. If this is not done the binary will not read the conf from /etc.

Comment (oschwald) [reply README.md]: Thanks. I understood that, but the concern is that less sophisticated users than yourself will follow the instructions without understanding the interactions between their install of `geoipupdate` and the packaged version (which they may be overwriting). I'd be happy to merge a variant on this that just explained what `--prefix` and `--sysconfdir` are and how to use them without the explicit command, but, as is, people will copy and paste it without thinking.

**Comment by oschwald** [comment]
I apologize. I didn't realize you addressed my comments. Thanks for your contribution!

**Comment by eRadical** [comment]
No need for apologies... thank you for your work on GeoIP.

**Pull request merged**